### DISCUSSION TITLE
Image projection from spotlight?

### DISCUSSION THREAD

**Discussion by ThomasCartier** [bevyengine/bevy#14476] [Q&A] [answered]
Hi,
Is it possible to project an image/video from a spotlight ?
Didn't see anything going that direction but maybe I missed something along the way.

Thanks

**Comment by bugsweeper** [comment, marked as answer]
I haven't seen mention of such a feature in the documentation either, but I do remember a [mention](https://bevyengine.org/news/bevy-0-10/#standardmaterial-blend-modes) of such a feature in the 0.10 release news. Take a close look at the animation of light passing through a colored mosaic. You will see that the light passing through glass of different colors is also colored. In the text of the chapter you can see a link to the [source code](https://github.com/coreh/bevy-demo-ruins) of this animation. In this way, you can put a transparent object with a texture on the way from your spotlight and broadcast the image as in slide projector.

**Reply by ThomasCartier** [reply]
Thanks again! I will code the plane as a child of the spotlight and be done with it.
